src/components/Sketches/components/EditSketchModal.js

```diff
@@ -224,24 +226,22 @@ class EditSketchModal extends React.Component {
               <Button
                 color="primary"
                 size="lg"
-                onClick={() => {
-                  this.setState({ onThumbnails: true });
-                }}
+                onClick={() => setOnThumbnails(true)}
```

Comment:
For this and the other `onClick` handlers, you should add an `onKeyDown` and `tabIndex` attribute